setup.py

```diff
@@ -81,7 +81,7 @@
             "pytest-asyncio==0.10.0",
             "pytest-cov>=2.0.0,<=2.9.0",
             "coverage>=4.0.3",
-            "pytest-docker-fixtures",
+            "pytest-docker-fixtures==1.3.11",
```

Comment:
Is this pin necessary?

Reply:
this is the version that was released with support for the memcached fixture

Reply:
Can we do `>=`?